src/chapters/setup-intro-data-types/exercises.rst

```diff
@@ -1,2 +1,152 @@
+      *Reopen last project on startup*. (For Windows users: *File >
+      Settings > Appearance & Behavior > System Settings*.) Close and
+      Reopen IntelliJ.
+
+#. From the “Welcome to IntelliJ” dialog, select *Check out from Version
+   Control > Git*
+   **TODO: Get screenshots of this process.**
+#. Click the button on the lower left corner of the dialog to configure your Github account.
+#. The first time you use GitHub within IntelliJ, you’ll have to
+   configure IntelliJ to use your GitHub account. We recommend taking
+   the extra step of authenticating using a token. This takes only one
+   brief extra step, and will prevent you from having to update IntelliJ
+   settings if you ever change your GitHub password.
+#. From the URL dropdown options, select your fork of ``java-web-dev-exercises``, 
+   along with an appropriate source destination directory (i.e. a folder where you’ve stored other projects for this class).
+#. When asked “Would you like to create an IDEA project…” select *Yes*, 
+   and then accept all of the default options that are presented.
+
+With that, you’re ready to go!
+
+Here’s a video walking through the last 4 steps above:
+
+.. raw:: html
+
+   <div class="youtube-wrapper">
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/OPCaYVXRm_c" frameborder="0" allowfullscreen>
+
+.. raw:: html
+
+   </iframe>
+
+.. raw:: html
+
+   </div>
+
+Troubleshooting
+---------------
+
+ClassNotFoundException
+^^^^^^^^^^^^^^^^^^^^^^
+
+If you experience ``java.lang.ClassNotFoundException`` when trying to
+run code after setting up the project, follow these steps: 
+
+#. Select *File > Close Project*.
+#. From the IntelliJ modal window, click the *X* next to ``java-web-dev-exercises`` in the left-hand pane.
+#. From the same modal window, select *Import Project* from the right-hand pane.
+#. Follow the steps that IntelliJ guides you through, accepting all defaults. When prompted to overwrite IntelliJ settings files, you confirm that you want to do so.
+
+
+Instructions
+------------
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project. Create a
+new class for each numbered exercise. You may name the classes whatever you like, but use
+proper :ref:`naming-conventions` and make sure that the file name matches the class name.
+
+Creating a Package and Classes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is how to create a new package to store these exercises, and how to
+create new classes within this package:
+
+#. Click on the folder ``src`` in the Project pane, then right click and
+   select *New* and then select *Package*.
+
+   .. figure:: figures/newPackage.png
+      :alt: New Package
+
+      New Package
+
+#. Name the package “exercises”.
+
+   .. figure:: figures/namePackage.png
+      :alt: Name Package
+
+      Name Package
+
+#. Then click on the newly-created folder ``exercises`` and right click
```

Comment:
Word removal suggestion.  How about, "Right-click on the newly created ``exercises`` folder.  Select *New* and then *Java Class*."